acceptance-tests/src/main/kotlin/org/ostelco/at/common/StripePayment.kt

```diff
@@ -43,6 +43,25 @@ object StripePayment {
         return token.card.id
     }
 
+    fun getDefaultSourceForCustomer(customerId: String) : String {
+
+        // https://stripe.com/docs/api/java#create_source
+        Stripe.apiKey = System.getenv("STRIPE_API_KEY")
+
+        val customer = Customer.retrieve(customerId)
+        return customer.defaultSource
+    }
+
+    fun getCustomerIdForEmail(email: String) : String {
+
+        // https://stripe.com/docs/api/java#create_card_token
+        Stripe.apiKey = System.getenv("STRIPE_API_KEY")
+
+        val customers = Customer.list(emptyMap()).data
```

Comment:
We are currently not storing the email with the customer id in Stripe. So how do this work ?